packages/ui-date-input/src/DateInput/index.js

```diff
@@ -257,6 +254,14 @@ class DateInput extends Component {
     children: null
   }
 
+  componentDidMount() {
+    this.props.makeStyles()
+  }
+
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
Parameters not needed 
```suggestion
  componentDidUpdate() {
```

Reply:
WebStorm complains if they are missing... that is why Matyi and me always add them :D